tests/unit/Grid.ts

```diff
@@ -0,0 +1,209 @@
+import * as registerSuite from 'intern!object';
+
+import harness, { Harness } from '@dojo/test-extras/harness';
+import { assignChildProperties, compareProperty } from '@dojo/test-extras/support/d';
+import { v, w } from '@dojo/widget-core/d';
+import RegistryHandler from '@dojo/widget-core/RegistryHandler';
+import WidgetRegistry from '@dojo/widget-core/WidgetRegistry';
+
+import Body from '../../src/Body';
+import Grid, { GridProperties } from '../../src/Grid';
+import Header from '../../src/Header';
+import { Column, ItemProperties, SortDetails } from '../../src/interfaces';
+import ArrayDataProvider from '../../src/providers/ArrayDataProvider';
+import * as css from '../../src/styles/grid.m.css';
+
+let widget: Harness<GridProperties, typeof Grid>;
```

Comment:
I think this declaration should be just before `registerSuite`, on two accounts:

* always declare constants first
* declare variables where they are first used (as opposed to our JS practice of declaring all vars at top to avoid hoisting confusion)